hasher-matcher-actioner/hmalib/common/tests/test_action_rule_evaluation.py

```diff
@@ -34,6 +34,7 @@ def test_get_action_labels(self):
 
         action_rules = [
             ActionRule(
+                enqueue_for_review_action_label.value,
```

Comment:
(Ignore if this is just temporary): Means you can only have one rule per action, is that correct? Two actions can't lead to a delete?

Reply:
@Dcallies This is just a simple test... you can definitely have another ActionRule in action_rules that has the same action label. The second rule just has to have a different name.